app/models/constants/states.rb

```diff
@@ -0,0 +1 @@
+STATES = ["AK","AL","AR","AS","AZ","CA","CO","CT","DC","DE","FL","GA","GU","HI","IA","ID","IL","IN","KS","KY","LA","MA","MD","ME","MI","MN","MO","MS","MT","NC","ND","NE","NH","NJ","NM","NV","NY","OH","OK","OR","PA","PR","RI","SC","SD","TN","TX","UT","VA","VI","VT","WA","WI","WV","WY"]
```

Comment:
Ack, I'm sorry, yesterday Casi was not thinking clearly. These constants should actually be part of your `State` class, so they belong in `/app/models/state.rb`. You should then be able to access them elsewhere as `State::STATES`.